src/libecalc/dto/result/tabular_time_series.py

```diff
@@ -18,7 +19,7 @@
 
 
 class TabularTimeSeries(ABC, EcalcResultBaseModel):
-    name: str
+    name: Optional[str] = None
```

Comment:
Given these changes, TabularTimeSeries shouldn't handle both `to_dataframe` and `resample`. Since name is something we need in `to_dataframe`. This change should include some refactoring to separate them. 

Instead of adding to this, can we move resample into its own class/function? So instead of calling result.resample(), we use Resampler.resample(result). At least that doesn't make to_dataframe more confusing.

Reply:
And put the resampler-function in EcalcBaseModel (dto.base)? I guess it is easier to have the resampler as a function in a class that all inherit from, compared to have a separate Resampler-class?

Reply:
Changed, name not optional.